src/client/components/clearable-input/clearable-input.mocha.tsx

```diff
@@ -25,8 +25,8 @@ import { renderIntoDocument } from "../../utils/test-utils";
 import { ClearableInput } from "./clearable-input";
 
 describe("ClearableInput", () => {
-  it("adds the correct class", () => {
-    var renderedComponent = renderIntoDocument(
+  it.skip("adds the correct class", () => {
```

Comment:
This is very stupid test. You can do better than this, my friend! Test if `className` is different for component with or without `value`. Or (and?) if given `className` is added to class list.